events/pony.txt

```diff
@@ -243,22 +243,22 @@ country_event = {
 }
 
 
-# #############################################################################################################
-
+##############################################################################################################
 
 # activates crystal cores
+# Crystal Empire independance events. Here it gains cores on its de jure territory.
```

Comment:
I didn't particularly look this through. Just applied your code and kept the current version commands.